packages/library/src/components/bal-select/value.utils.ts

```diff
@@ -0,0 +1,44 @@
+import { BalOptionController } from '../bal-select-option/bal-select-option'
+import { findOptionByLabel, findOptionByValue } from './option.utils'
+
+export function removeValue(value: string[], valueToRemove: string): string[] {
+  const copyOfValues = []
+  for (let index = 0; index < value.length; index++) {
+    if (value[index] !== valueToRemove) {
+      copyOfValues.push(value[index])
+    }
+  }
+  return copyOfValues
+}
+
+export function addValue(values: string[], valueToAdd: string, hasMultipleValue: boolean) {
+  if (hasMultipleValue) {
+    if (values.includes(valueToAdd)) {
+      return removeValue(values, valueToAdd)
```

Comment:
Strange as the method is called `addValue` but you remove values here?
Is it a toggle?